### PR TITLE
Use default port in the chart of Scalar DB and Auditor

### PR DESCRIPTION
**Pull request by kota2and3kan**
This PR updates Scalar DB and Auditor charts to use default ports (Scalar DB -> 60051 / Auditor -> 40051, 40052).

Please take a look!

### CODE REVIEW

File: charts/scalardb/templates/scalardb/deployment.yaml [hunk 1]
```diff
@@ -72,15 +72,13 @@ spec:
             value: "{{ .Values.scalardb.storageConfiguration.storage }}"
           - name: SCALAR_DB_LOG_LEVEL
             value: "{{ .Values.scalardb.storageConfiguration.dbLogLevel }}"
-          - name: SCALAR_DB_SERVER_PORT
-            value: "50051"
```
Comment (kota2and3kan) [inline charts/scalardb/templates/scalardb/deployment.yaml:76]: When we use default port (60051) in the Scalar DB Helm Charts, we can remove this hard-coded value from manifest file.

File: charts/scalardb/values.yaml [hunk 2]
```diff
@@ -19,7 +19,7 @@ envoy:
     # -- admin log path
     adminAccessLogPath: /dev/stdout
     # -- list of service name and port
-    serviceListeners: scalardb-service:50051,scalardb-privileged:50052
+    serviceListeners: scalardb-service:60051
```
Comment (kota2and3kan) [inline charts/scalardb/values.yaml:22]: Scalar DB doesn't have **privileged port** like Scalar DL.
So, we need the `scalardb-service:60051` only.

**Review by ymorimo — APPROVED** [review]
LGTM! Thank you!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**